Source/santad/EventProviders/SNTEndpointSecurityClient.mm

```diff
@@ -92,7 +92,7 @@ - (instancetype)initWithESAPI:(std::shared_ptr<EndpointSecurityAPI>)esApi
     _notifyQueue = dispatch_queue_create(
       "com.google.santa.daemon.notify_queue",
       dispatch_queue_attr_make_with_qos_class(DISPATCH_QUEUE_CONCURRENT_WITH_AUTORELEASE_POOL,
-                                              QOS_CLASS_BACKGROUND, 0));
+                                              QOS_CLASS_DEFAULT, 0));
```

Comment:
Documentation states that `QOS_CLASS_DEFAULT` shouldn't be directly specified. We should change this instead to just call:
```
 _notifyQueue = dispatch_queue_create(
      "com.google.santa.daemon.notify_queue",
      DISPATCH_QUEUE_CONCURRENT_WITH_AUTORELEASE_POOL);
```

https://github.com/apple-oss-distributions/libpthread/blob/d8c4e3c212553d3e0f5d76bb7d45a8acd61302dc/include/sys/qos.h#L78-L86

Reply:
Done